coremltools/converters/mil/mil/passes/defs/optimize_tensor_operation.py

```diff
@@ -839,6 +839,19 @@ def apply(self, prog):
         for f in prog.functions.values():
             self._reflection_padding_block(f)
 
+    @staticmethod
+    def _resolve_end_mask(slice_op):
```

Comment:
I think it would be a little better for this to be an inner method, i.e. define `_resolve_end_mask` inside (and at the start of) `_match_pattern`.